cpp/test/rank.cc

```diff
@@ -51,3 +51,18 @@ TEST(RankTest, TestRankDescription) {
   ASSERT_FALSE(a.isFlush());
   ASSERT_TRUE(b.isFlush());
 }
+
+TEST(RankTestStraights,TestRankDescriptionStraights){
```

Comment:
The first argument is the test group. I’d still like it to be RankTest so that it would be in a group with other similar test cases. You’ve corrected the second argument, which is very good.

In addition, please add a space after the comma, between the two arguments.